src/libraries/System.Formats.Tar/tests/TarEntry/TarEntryPax.Tests.cs

```diff
@@ -36,6 +36,224 @@ public void Constructor_UnsupportedEntryTypes()
             Assert.Throws<InvalidOperationException>(() => new PaxTarEntry(TarEntryType.GlobalExtendedAttributes, InitialEntryName));
         }
 
+        [Fact]
+        public void Constructor_ConversionFromV7()
+        {
+            V7TarEntry v7 = new V7TarEntry(TarEntryType.V7RegularFile, InitialEntryName);
+            PaxTarEntry convertedV7 = new PaxTarEntry(other: v7);
+
+            Assert.Equal(TarEntryType.RegularFile, convertedV7.EntryType);
+            Assert.Equal(InitialEntryName, convertedV7.Name);
+        }
+
+        [Fact]
+        public void Constructor_ConversionFromUstar()
+        {
+            UstarTarEntry ustar = new UstarTarEntry(TarEntryType.RegularFile, InitialEntryName);
+            PaxTarEntry convertedUstar = new PaxTarEntry(other: ustar);
+
+            Assert.Equal(TarEntryType.RegularFile, convertedUstar.EntryType);
+            Assert.Equal(InitialEntryName, convertedUstar.Name);
+        }
+
+        [Fact]
+        public void Constructor_ConversionFromGnu()
+        {
+            GnuTarEntry gnu = new GnuTarEntry(TarEntryType.RegularFile, InitialEntryName);
+            PaxTarEntry convertedGnu = new PaxTarEntry(other: gnu);
+
+            Assert.Equal(TarEntryType.RegularFile, convertedGnu.EntryType);
+            Assert.Equal(InitialEntryName, convertedGnu.Name);
+        }
```

Comment:
I don't remember seeing any of these tests talk about the content stream of the entry.  Assuming the user can bind a stream, whatever can be asserted should be.  (If it's `Assert.Same(converted.Stream, orig.Stream)` then that says enough... assuming, of course, that they're not just both null...)

Reply:
I added an `Assert.Same` for the stream.